ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java

```diff
@@ -175,8 +172,8 @@ public TaskResult run() {
       return new TaskResult(TaskResult.Status.FAILED, "Exception thrown");
     } finally {
       if (clusterMapConfig.clustermapEnableContainerDeletionAggregation && callback != null
```

Comment:
I'll remove this part of logic after your changes since it won't be used any more cause we leverage orca job to do it.

Reply:
sure, let's do this in next PR.